source/guides/models/repositories.md

```diff
@@ -95,6 +95,7 @@ class BookRepository < Hanami::Repository
       .where(author_id: author.id)
       .order(:published_at)
       .limit(limit)
+      .to_a
   end
 end
```

Comment:
And we need this method here? Maybe is better to add a description why we call `#to_a`?